### PR TITLE
update to 1.15.2

### PR DESCRIPTION
**Pull request by doits**

### CODE REVIEW

File: lib/uri-js-rails/version.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module Uri
   module Js
     module Rails
-      VERSION = "1.14.2.1"
+      VERSION = "1.15.2.1"
```
Comment (PikachuEXE) [inline lib/uri-js-rails/version.rb]: I don't think a PR should change this...

**Comment by PikachuEXE** [comment]
Please fix the specified problem and reply.
There is no notification for a push.

**Comment by doits** [comment]
Removed version change

**Comment by PikachuEXE** [comment]
Thanks!
Will release a new version soon

**Pull request merged**